src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx

```diff
@@ -114,36 +115,53 @@ export default function SimilarStatementsSuggestion({
 			) : (
 				<main className='similarities'>
 					<header className='similarities__header'>
-						{currentStep === 2 ? (
-							<div className='similarities__header__top'>
+						{currentStep === 1 ? (
+							<div className='similarities__header__top__stepTwo'>
 								<button
-									className='similarities__header__top__closeButton'
+									className='similarities__header__top__stepTwo__closeButtonStepTwo'
 									onClick={() => setShowModal(false)}
 								>
-									<ArrowLeftIcon />
+									<CloseIcon />
 								</button>
-								<img src={illustration02} alt='Similarities illustration' />
+								<img src={illustration01}
+									alt='Similarities illustration'
+									style={{ width: '8rem' }} />
 							</div>
```

Comment:
Consider refactoring this into a separate function that handles rendering based on the current step. The current implementation is difficult to read and maintain.

Reply:
Do the recent changes i made answer your request?